PhysicsTools/NanoAOD/python/btagWeightTable_cff.py

```diff
@@ -1,7 +1,5 @@
 import FWCore.ParameterSet.Config as cms
 from PhysicsTools.NanoAOD.common_cff import *
-from PhysicsTools.NanoAOD.nano_eras_cff import *
```

Comment:
We discussed removing these branches altogether, perhaps it could be done in this PR? (though unfortunately we have to keep the producer code for V10 compat)

Reply:
Reminder